views/docs/index.pug

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+html(lang="en")
+ include partials/head
+
+body(data-spy="scroll" data-target="#scroll-spy")
+  div.hero-wrapper
+    div.container.hero
+      div.row
+        div.col-md-12
+          h1 Remember This? 
+          p.lead An API to remind you of all the things that made your childhood amazing.
+
+  div.container
+    div.row
+      include partials/side-nav
+      div.col-md-9
+        section#general
+          h2 Overview
+          p.lead This API provides data to evoke visceral pop-culture memories.
+          p Documentation for this API resides at the server's root directory. <br> 
+            small (You are here. Congratulations!)
+          p See below for available routes and what you can do.
+          p Enjoy!
+        
+        h2#get-overview.overview-heading GET
+        include partials/sections/get-all
+        include partials/sections/get-slug
+        include partials/sections/get-category
+        include partials/sections/get-decade
+
+        h2#post-overview.overview-heading POST
+        include partials/sections/post-root
```

Comment:
Remember, APIs are for machines, a helpful response body is the new record with the id that can be parsed and reused by the code talking to your API.

![image](https://user-images.githubusercontent.com/5775716/83890287-91c74c80-a719-11ea-8840-5ccd750af8b2.png)